File: src/chameleon.ml
#load "str.cma"
let s="DATA";;{{{chameleon:Random.self_init()}}}
let l=Array.of_list(Str.split(Str.regexp"\t")s) and i={{{chameleon:FIELDCOUNT*Random.int LANGCOUNT}}}{{{ouroboros:ENTRYINDEX}}};;print_string(l.(i)^(String.escaped s)^l.(i+1))

File: src/chameleon.fs
let S="DATA"
let L=S.Split '\t'
let I={{{chameleon:System.Random().Next(LANGCOUNT)*FIELDCOUNT}}}{{{ouroboros:ENTRYINDEX}}}
printf"%s"(L.[I]+S.Replace("\\",@"\\").Replace("\t",@"\t").Replace("\n",@"\n").Replace("\"",@"\""")+L.[I+1])
